playground/ssr-react/__tests__/ssr-react.spec.ts

```diff
@@ -50,7 +50,8 @@ test('/', async () => {
 })
 
 test('hmr', async () => {
-  await page.goto(url)
+  await untilBrowserLogAfter(() => page.goto(url), 'hydrated')
```

Comment:
This test should wait for hydration because HMR won't work before hydration like other tests.